helga_jeopardy.py

```diff
@@ -146,17 +171,41 @@ def reveal_answer(client, channel, question_id, answer, mongo_db=db.jeopardy):
         logger.debug('not active question, someone must have answered it! Good Show!')
         return
 
-    client.msg(channel, u'the correct answer is: {}'.format(answer))
+    if random is True:
```

Comment:
While this is not incorrect, the "Pythonic" way to test for `True` is just to say `if some_bool:`. Similarly, to test for `False`, it is idiomatic to say `if not some_bool:`.

```suggestion
    if random:
```